node_modules/oae-principals/lib/api.user.js

```diff
@@ -77,14 +77,21 @@ var registerFullUserProfileDecorator = module.exports.registerFullUserProfileDec
  * authentication strategy with this user record as the user would otherwise have no means
  * of logging onto the system.
  *
+ * If an email address is specified, it's assumed that you have ensured that no other user
+ * account with this email address exists.
+ *
  * @param  {Context}   ctx                      Standard context object containing the current user and the current tenant
+ * @param  {String}    [tenantAlias]            The alias of the tenant on which to create the user. If left undefined, the current tenant's alias will be used
  * @param  {String}    displayName              The display name for the user
  * @param  {Object}    [opts]                   Optional parameters for the user
  * @param  {String}    [opts.visibility]        The visibility of the user. One of AuthzConstants.visibility
  * @param  {String}    [opts.locale]            The locale for the user
  * @param  {String}    [opts.publicAlias]       The name to show when the user is inaccessible to a user
  * @param  {Boolean}   [opts.acceptedTC]        Whether or not the user has accepted the Terms & Conditions
+ * @param  {String}    [opts.email]             The email addres for the user
```

Comment:
It is optional though, as we need to be able to create the actual user account (for things like SSO)